packages/odyssey-react-mui/src/test-selectors/sanityChecks.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// Code modified from: https://github.com/testing-library/jest-dom/blob/main/src/utils.js
+
+export class ElementError extends Error {
+  constructor(message: string, element: HTMLElement) {
+    super(message);
+
+    this.name = "ElementError";
+
+    console.error(element);
```

Comment:
Logging the error in the constructor seems a bit strange, it might end up producing the log somewhere unexpected if the Error is created and then caught somewhere without immediately throwing. Probably doesn't matter much though.

Reply:
I didn't know a better way of doing this. When you throw an error, there's no way to pass an object along with the error; it always gets stringified, so I just logged it when the error came up.

Unless I'm wrong, I don't know if there's a method on the error like `thrown() {}` where I can log the HTML element.